### PR TITLE
Employees seeds

### PR DESCRIPTION
**Pull request by vladfreel**
Add seeds to Employee

### CODE REVIEW

File: db/migrate/20180601074919_add_skype_to_user.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddSkypeToUser < ActiveRecord::Migration[5.2]
```
Comment (polyakovigor) [inline db/migrate/20180601074919_add_skype_to_user.rb]: AddSkypeToEmployee

**Comment by drkmen** [comment]
Please rename migration and squash commits

**Comment by drkmen** [comment]
PS: i removed commits with seed file. You delete it, but it still was in 2 commits so every one can look at.

File: db/seeds.rb [hunk 2]
```diff
@@ -1,7 +1,193 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+# Менеджеры
```
Comment (Tivgres) [inline db/seeds.rb]: Need a squash, important.

**Comment by vladfreel** [comment]
Squash commits and resolve conflicts

**Comment by drkmen** [comment]
Why did you delete migration?

**Comment by drkmen** [comment]
Please squash commits!

**Pull request merged**